rts/lra/lra-coordinator/src/test/java/org/jboss/narayana/rts/lra/test/LRASmoke.java

```diff
@@ -41,7 +41,7 @@
 
 @RunWith(Arquillian.class)
 @RunAsClient
-public class LRASmokeIT {
+public class LRASmoke {
```

Comment:
I'm worry this will not work if not set up in pom.xml. Failsafe plugin by default run the IT suffixed classes, what I know.